### ISSUE TITLE
OpenMPI 4.1.5

### ISSUE THREAD

**Issue by bdevcich-hpe** [kubeflow/mpi-operator#588]
Any plans to update to the latest stable version of OpenMPI?

**Comment by alculquicondor** [comment]
It should just work. Feel free to send a PR.

**Comment by tenzen-y** [comment]
I see.

> Any concerns with building openmpi from source?

I want to avoid building the OpenMPI to avoid increasing maintenance costs.

> It looks like bookworm has 4.1.4: https://packages.debian.org/bookworm/openmpi-bin

Actually, we already have a PR to update the Debian version, although there are unresolved issues: https://github.com/kubeflow/mpi-operator/pull/573

Can you try to update the Debian version instead of building OpenMPI?

WDYT? @alculquicondor @terrytangyuan

**Comment by alculquicondor** [comment]
Yes, prefer to update the debian version.

**Comment by terrytangyuan** [comment]
Agreed. We should not build it from source

**Comment by bdevcich-hpe** [comment]
Makes sense. It adds some complexity. I think 4.1.4 (that comes with bookwork) will be fine as it contains the fix that we're interested in.

So the path forward is to get traction on #573 ?

**Comment by tenzen-y** [comment]
> Thanks.
> 
> Makes sense. It adds some complexity. I think 4.1.4 (that comes with bookworm) will be fine as it contains the fix that we're interested in.
> 
> So the path forward is to get traction on #573 ?

I think you can open the new PR :)

**Comment by alculquicondor** [comment]
In any case, you should be able to run any version of OpenMPI with the operator. You can build your containers against bookworm (or any other distro).

**Comment by abeltre1** [comment]
@bdevcich @tenzen-y is there a reason why we do not make the install from source? 
- [OMPI](https://www.open-mpi.org/software/ompi/v5.0/)
